src/iodev.h

```diff
@@ -11,6 +11,12 @@ typedef enum _iodev_id_t {
     IODEV_FB,
     IODEV_USB0,
     IODEV_USB1,
+    IODEV_USB2,
```

Comment:
And then if we make USB IODEVs the last in the list, we don't even have to actually list them all in the enum - just USB0 and MAX, with everything in between implied.

(Though we probably still want to list them in the python just so IntEnum doesn't complain about missing enum values)

Then USB_INSTANCES can be defined here (perhaps as USB_IODEV_COUNT instead) and usb.c can use that instead.